core/dynlib/lib_js.odin

```diff
@@ -0,0 +1,15 @@
+//+build js
+//+private
+package dynlib
+
+_load_library :: proc(path: string, global_symbols := false) -> (Library, bool) {
+	panic("core:dynlib not supported by JS target")
+}
+
+_unload_library :: proc(library: Library) -> bool {
+	panic("core:dynlib not supported by JS target")
+}
+
+_symbol_address :: proc(library: Library, symbol: string) -> (ptr: rawptr, found: bool) {
+	panic("core:dynlib not supported by JS target")
+}
```

Comment:
Would this be better to just return `false` for all of these?

Reply:
Maybe, but if not, then replace `panic("...")` with `unimplemented("...")`.

Reply:
I didn't know `unimplemented` exists. What is the difference between that and `panic`? But yeah maybe for the dynlib stuff a false could be nicer to use than straight busting your program. I'll update it

Reply:
It's still a `panic`, only it makes it easier to search for things that are `unimplemented`.

Reply:
Good call, i'll update my pr with your suggestion in a few hours